static/js/init.js

```diff
@@ -114,6 +115,9 @@ $(document).ready(function() {
     new Search($(this));
   });
 
+  // Initialize table of contents
+  new toc.TOC('.js-toc');
```

Comment:
Good catch. Yeah, some of our components take a selector string, some take an actual $ element.